### PR TITLE
Change README.md

### PR DESCRIPTION
**Pull request by gjcarew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# HELLO 
+# HELLO
+## Hello to you as well
```
Comment (eriggs0207) [inline README.md:2]: Does this show up?

Comment (gjcarew) [reply README.md:2]: > 

yes it does

Comment (eriggs0207) [reply README.md:2]: Does this show up?

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# HELLO 
+# HELLO
```
Comment (gjcarew) [inline README.md:1]: Can you see this?

**Review by gjcarew — COMMENTED** [review]
I'm doing the same as you

File: README.md (hunk 1 above)
Comment (gjcarew) [reply README.md:2]: It shows up for me

File: README.md (hunk 2 above)
Comment (gjcarew) [inline README.md:1]: this is testing a review # 3

**Review by gjcarew — COMMENTED** [review]
This has been reviewed

**Pull request merged**